### ISSUE TITLE
Marti issue

### ISSUE THREAD

**Issue by trulpen** [triplea-game/triplea#10096]
I don't seem to get almost any dicemails from my A&A-games. I presume you're aware of this problem, but post this issue just in case.

**Comment by beelee1** [comment]
yea it's discussed some here https://forums.triplea-game.org/topic/3061/marti-issues
I thought Roi made a fix ( not added to triplea yet ), but I may have that confused with something else.

**Comment by panther2** [comment]
@beelee1
No, there is no fix yet. It is not even proven that Marti code /TripleA is the culprit.

**Comment by ron-murhammer** [comment]
@panther2 Have you yourself been able to reproduce this? I use a @yahoo.com email address for marti and still receive the dice emails without issue though I don't check every single one so I guess its possible that some are missing/failing.

**Comment by panther2** [comment]
@ron-murhammer 
I have been able to reproduce this issue with different Google mail-addresses, indeed. 

To me the issue has different occurrences:
- I don't receive any mail at all
- I receive a fraction of related mails
- I receive mails automatically sorted in to my spam folder by Google

**Comment by frigoref** [comment]
@ron-murhammer Are you going to cover this one?
@panther2 If ron is not covering this, could you please provide a proper step-by-step guide to reproduce the issue and a clear description? So far I understand that mails to Google-addresses are not always received. What is Marti?

**Comment by beelee1** [comment]
@frigoref marti is the dice server that triplea uses to generate the dice rolls/emails. Talked about some here https://forums.triplea-game.org/topic/3061/marti-issues?page=1

**Comment by panther2** [comment]
@frigoref 

As @beelee1 said MARTI is the external dice server used for PBF/PBEM

1. Configure a PBF/PBEM game to use the MARTI dice server with a GMail-address.
2. When playing perform combat moves to create battles.
3. Execute the battles.
4. Look in your Inbox/Spam folders

Note these occurences of the issue:
- I don't receive any MARTI mail at all
- I receive a fraction of related MARTI mails
- I receive MARTI mails automatically sorted in to my spam folder by Google
